spec/license_spec.rb

```diff
@@ -27,6 +27,22 @@
           expect(approved_licenses).to include(id), "See https://git.io/vzCTV."
         end
       end
+
+      context "minimum permissions" do
+        permissions = license["permitted"]
+        it "should allow commercial use" do
+          expect(permissions).to include("commercial-use")
+        end
+        it "should allow modification" do
```

Comment:
Technically, technically, Ruby style says the inner blocks should have whitepace around them (e.g., lines 36, 39, 42), like https://github.com/github/choosealicense.com/blob/gh-pages/spec/license_category_spec.rb#L21-L28.